history/characters/340000_night_elf.txt

```diff
@@ -328,7 +328,7 @@
 				limit = { NOT = { current_date >= 604.1.1 } }
 				imprison = {
 					target = character:340009	#Illidan
-					type = dungeon
+					type = house_arrest			# Hotfix for poor health in dungeon
```

Comment:
I think it doesn't fit. Better give him a health bonus: Whole of Body trait, etc.

Reply:
Decided to keep House Arrest as vanilla health modifiers insufficient to counter long term dungeon health malus.